lnrpc/walletrpc/walletkit_server.go

```diff
@@ -268,7 +270,19 @@ func (w *WalletKit) PublishTransaction(ctx context.Context,
 		return nil, err
 	}
 
-	err := w.cfg.Wallet.PublishTransaction(tx)
+	if len(req.Label) > wtxmgr.TxLabelLimit {
+		return nil, fmt.Errorf("label length: %v exceeds "+
+			"limit of %v", len(req.Label), wtxmgr.TxLabelLimit)
+	}
+
+	// If no label was provided by the user, add the generic user
+	// send label.
+	label := req.Label
+	if len(label) == 0 {
+		label = labels.UserSend
```

Comment:
This may not be the appropriate label for higher-level applications using this API as they won't be user initiated in a sense.

Reply:
Renamed this to `API`, could also go with `External`?

Reply:
I prefer the latter but either works.